DB/api/query_db.py

```diff
@@ -53,8 +56,17 @@ def query(parsed_url):
         list_format = parsed_url['list_format']
     if 'events' in parsed_url.keys():
         events = parsed_url['events']
+    if 'total_energy' in parsed_url.keys():
+        total_energy = parsed_url['total_energy']
+
+    if classify:
+        if device is not None and start_time is not None:
+            model = RMF.RandomForestModel.get_model()
+            classification = model.classify(time=start_time, serial=device_id, panel=device)
+            return format_data(['data'], classification, json=True)
+        raise Exception("Received malformed URL data: missing start_time")
```

Comment:
Because the end-time is determined by the window size and is generated internally inside the query execute method based on the window size of the model it's trying to classify with.